profiles/common/modules/features/nexteuropa_newsroom/theme/newsroom-block-item.tpl.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @file
+ * Agenda.
+ */
+?>
+<?php if (count($items) > 0) : ?>
+  <div class="view view-newsroom-page-content">
+    <div class="view-content">
+      <?php foreach ($items as $item) : ?>
```

Comment:
put this foreach into a template/theme/preprocess function and reduce the mix of templating with business logic.